book/accessibility.md

```diff
@@ -322,12 +322,12 @@ class BlockLayout:
 	# ....
     def word(self, node, word):
     	# ...
-        size = dpx(float(node.style["font-size"][:-2]),
+        size = dpx(float(node.style["font-size"][:-2]) * 0.75,
```

Comment:
Was the previous code here and in various other cases just wrong? In chapter 6 we multiply by 0.75, but I guess that was incorrectly lost in later chapters?

Reply:
Indeed - skia.Font expects points, not pixels.

Reply:
Yes, it really looks like!

Reply:
It was hard to notice because we always test on the book website, which specifies its own font size in CSS, and then I guess we'd just never compared Skia and Tk renderings to notice the font size difference.